administrator/components/com_installer/tmpl/manage/default.php

```diff
@@ -106,7 +106,29 @@
 									<?php echo $item->type_translated; ?>
 								</td>
 								<td class="hidden-sm-down text-center">
-									<?php echo @$item->version != '' ? $item->version : '&#160;'; ?>
+									<?php if($item->version != ''): ?>
```

Comment:
You need a space after the if statement for codestyle. Check the rest of the code as well.